trio/_core/_tests/test_asyncgen.py

```diff
@@ -44,14 +43,14 @@ async def async_main() -> None:
         with pytest.warns(
             ResourceWarning, match="Async generator.*collected before.*exhausted"
         ):
-            assert 42 == await example("abandoned").asend(None)
+            assert await example("abandoned").asend(None) == 42
```

Comment:
I don't get how this simplifies anything?

Reply:
I like this as a general style thing, though I think the developers of this repo already adheres to it decently well. E.g. code like
```python
if (a == 3 || 4 == b):
  ...
```
is very silly, and in spirit feels like something `Black` would enforce - except they are explicitly AST-neutral. So I think there's good that there's a tool that enforces it.

Reply:
Oh, I actually very much prefer the changes to this set of comparisons. Because they go against the standard, I was thinking maybe it's because the developer wants to place an emphasis on the left term because the right term is constant ... but no the literal is just a random magic number and the expression on the right hand side very much does change.